src/frequenz/client/electricity_trading/__init__.py

```diff
@@ -1,7 +1,131 @@
 # License: MIT
 # Copyright © 2024 Frequenz Energy-as-a-Service GmbH
 
-"""Electricity Trading API client for Python."""
+"""
+Electricity Trading API client for Python.
+
+## Frequenz Electricity Trading API Client
+
+This module provides an easy-to-use Python interface to interact with the Frequenz Electricity
+Trading API. It allows you to create orders, manage market data, and interact with the electricity
+trading ecosystem.
+
+### Features
+
+- **Create and manage gridpool orders**: Place new orders, update existing ones, and cancel orders
+when necessary.
+- **Stream live data**: Get real-time updates on market data, including order books, trades,
+and market prices.
+- **Retrieve historical data**: Access historical data on market trades.
+
+### Installation
+
+You can install the Frequenz Electricity Trading API client via pip. Replace `VERSION`
+with the specific version you wish to install.
+
+# Choose the version you want to install
+```python
+VERSION=0.2.3
+pip install frequenz-client-electricity-trading==$VERSION
+```
+
+### Initialization
+
+First, initialize the client with the appropriate server URL and API key.
+
+???+ example "Initialize the client"
+
+    ```python
+    from frequenz.client.electricity_trading import Client
+
+    # Change server address if needed
```

Comment:
I think all these code below should live within an async function, otherwise the client won't work if it gets created outside the async loop